config/development/clinc_dummy/conf_bma.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "CLINC Dummy",
```

Can we give a different name so that it's clear which cache is which? I don't know what "bma" means, so I don't know what case to use. That?
```suggestion
  "name": "CLINC Dummy BMA",
```

---

Good catch!